copilot-chat.el

```diff
@@ -3,7 +3,7 @@
 ;; Copyright (C) 2024  copilot-chat maintainers
 
 ;; Author: cedric.chepied <cedric.chepied@gmail.com>
-;; Version: 2.0.1
+;; Version: 2.1.0
```

Comment:
I forgot one thing, it would be better to raise the major version.
You changed the name of the customization variable, it is no longer compatible.

Reply:
Done !